.tool-versions

```diff
@@ -0,0 +1,5 @@
+# For use with ASDF: https://asdf-vm.com/
```

Comment:
Hmm, I'm not fan of optional tooling configuration inside the repository, as they are not as commonly used (lack of standard). This is why there is a `Dockerfile` and CI, which enforces the commonly agreed versions where builds etc must pass.

 Perhaps we could add a CI job instead which use specific versions of tooling then. I can take care of this. But CI should also be moved to GH actions actually... :smile:

Reply:
I suspected that it might not be accepted :( It just made my life a bit easier,
a) because it has all tool versions in one place, instead of them being scattered around config files, and 
b) because we use asdf+direnv that allows us to switch between projects easily.
So whenever I change into this project's folder, I have the right version of the tools in $PATH.

Sigh, I'll remove it.

Reply:
Perhaps you can keep the file for you locally, but add it to [git global ignore config](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)?

Reply:
Yes, that's what I am planning to do, only on a local level (`.git/info/exclude`).

Reply:
Thanks, I appreciate the understanding :)